client/me/purchases/manage-purchase/test/payment-info-block.js

```diff
@@ -66,9 +66,11 @@ describe( 'PaymentInfoBlock', () => {
 		describe( 'when the purchase a non-rechargable payment method', () => {
 			const purchase = { expiryStatus, payment: { type: 'ideal' }, isRechargeable: false };
```

Comment:
I think you'll need to add `isAutoRenewEnabled: true` to make these tests pass. 🤔 I'm fairly certain it's always going to be set because of this line:

https://github.com/Automattic/wp-calypso/blob/4b04c24154e527cd8440cc1f70fdc1e81a4941de/client/lib/purchases/assembler.ts#L99

Reply:
That helped, I set the Purchase object like this and it worked:

```
const purchase = {
				expiryStatus,
				payment: { type: 'ideal' },
				isRechargeable: false,
				isAutoRenewEnabled: true,
			};
```